### ISSUE TITLE
[BUG] The pretrained model available on Zenodo ([10829675](https://zenodo.org/records/10829675)) appears to contain only `fold_0`

### ISSUE THREAD

**Issue by tcrnscw** [gaudot/SlicerDentalSegmentator#35]
Hello, I am trying to use the DentalSegmentator model from Zenodo to do batch processing through nnunet directly. When running inference with this model, segmentation quality is poor — specifically, label mixing between the mandible and upper skull/maxilla classes, with large regions assigned to the wrong structure. 

From what I understand the nnU-Net inference should ensemble predictions across all 5 folds - is this correct? Only `fold_0` is present in the Zenodo deposit. Running with `-f 0` explicitly produces the segmentation, but with significant label confusion between mandible and skull classes (see photo).

Are the remaining folds (fold_1 through fold_4) available? If so, would it be possible to upload them to Zenodo or share them another way? If only fold 0 is necessary, I assume this is a bug on my end - apologies!

Thank you for making this tool available — it's a genuinely useful resource for the community!

<img width="374" height="439" alt="Image" src="https://github.com/user-attachments/assets/0ba196c2-f9b8-4517-87b8-652723187bfc" />

**Comment by Thibault-Pelletier** [comment]
Hi @tcrnscw,

Thank you for your message.

Regarding the weights, both the Zenodo and the Github weights only contain fold_0 and the Slicer module indeed only executes on fold_0. Note that the weights [available on Github](https://github.com/gaudot/SlicerDentalSegmentator/releases/tag/v1.0.0-alpha) may be more up to date than the ones on Zenodo.

I will let @GauthierDot, one of the paper's authors, answer on why we only use fold_0.

Best,
Thibault

**Comment by GauthierDot** [comment]
HI @tcrnscw 

Thank you for your interest. As detailed in the paper, we chose to train only one fold (fold_0) and assessed the segmentation performance on external test data. We did not performed any ensemble predictions across the 5 folds.

The weights shared on Zenodo and Github should be the same and you should get the same results using 3DSlicer extension or command line interface (except a postprocessing applied on 3DSlicer to remove very small isolated islands).

About the error you're showing on your screenshot, do you also get it using 3DSlicer extension?

Best
Gauthier
